PRIVACY.md

```diff
@@ -1,134 +1,154 @@
+
+SimpleX Chat software is built on top of SimpleX messaging and application protocols, based on a new message routing protocol allowing to establish private connections without having any kind of addresses or other identifiers assigned to its users - it does not use emails, phone numbers, usernames, identity keys or any other user profile identifiers to pass messages between the user applications.
+
+SimpleX Chat software is similar in its design approach to email clients and browsers - it allows you to have full control of your data and freely choose the relay server providers, in the same way you choose which website or mail provider to use, or use your own relay servers, simply by changing the configuration of the software. The only current restriction to that is Apple push notifications - at the moment they can only be delivered via the preset servers that we operate, as explained below. We are exploring the solutions to deliver push notifications to iOS devices via other providers or users' own servers.
+
+While SimpleX Chat Ltd is not a communication service provider, and provide any preset relays "as is", as experimental, without any guarantees of availability or data retention, we are committed to maintain a high level of availability, reliability and security of these preset relays. We will be adding alternative preset infrastructure providers to the software in the near future, and you will continue to be able to use any other providers or your own servers.
 
-SimpleX Chat is built on top of SimpleX messaging and application platform that uses a new message routing protocol allowing to establish private connections without having any kind of addresses that identify its users - we don't use emails, phone numbers, usernames, identity keys or any other user identifiers to pass messages between the users.
+We see users and data sovereignty, and device and provider portability as critically important properties for privacy and security of the communications.
 
-SimpleX Chat security assessment was done in October 2022 by [Trail of Bits](https://www.trailofbits.com/about), and most fixes were released in v4.2.0 – see [the announcement](./blog/20221108-simplex-chat-v4.2-security-audit-new-website.md).
+SimpleX Chat security assessment was done in October 2022 by [Trail of Bits](https://www.trailofbits.com/about), and most fixes were released in v4.2 – see [the announcement](./blog/20221108-simplex-chat-v4.2-security-audit-new-website.md).
 
-### Information you provide
+### Your information
 
 #### User profiles
 
-We do not store user profiles. The profile you create in the app is local to your device.
+Servers used by SimpleX Chat apps do not create, store or identify user profiles. The profiles you can create in the app are local to your device, and can be removed at any time.
```

Comment:
Can you please mention SimpleX addresses, 1-time invitation links and the user's "contact list" which are currently the SimpleX addresses of their contacts (if they get it from them) and/or the conversations itself if they are based on 1-time link. 

What part of these three things SMP servers store and needed for clients to reconnect in case they reinstall client app or migrate phones or their orig SMPs go offline etc.